browser.js

```diff
@@ -0,0 +1,35 @@
+/* globals document, ImageData */
+
+const parseFont = require('./lib/parse-font')
+
+exports.parseFont = parseFont
+
+exports.createCanvas = function (width, height) {
+  return Object.assign(document.createElement('canvas'), { width, height })
+}
+
+exports.createImageData = function (array, width, height) {
+  // Browser implementation of ImageData looks at the number of arguments passed
+  switch (arguments.length) {
+    case 0: return new ImageData()
+    case 1: return new ImageData(array)
+    case 2: return new ImageData(array, width)
+    default: return new ImageData(array, width, height)
```

Comment:
Do cases 0 and 1 exist? [MDN only documents](https://developer.mozilla.org/en-US/docs/Web/API/ImageData/ImageData) `new ImageData(arr, w, h)` and `ImageData(w, h)`.

Also, would this be the same as `return new ImageData(...arguments)`?